target/linux/ramips/image/mt7620.mk

```diff
@@ -828,6 +828,19 @@ define Device/tplink_archer-c2-v1
 endef
 TARGET_DEVICES += tplink_archer-c2-v1
 
+define Device/tplink_archer-c5-v4
+  $(Device/Archer)
+  TPLINK_FLASHLAYOUT := 8Mmtk
+  TPLINK_HWID := 0x04DA857C
+  TPLINK_HWREV := 0x0C000600
+  TPLINK_HWREVADD := 0x04000000
+  IMAGES += factory.bin
+  DEVICE_MODEL := Archer C5
+  DEVICE_VARIANT := v4
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-usb-ledtrig-usbport kmod-mt76x2 kmod-switch-rtl8367s
+endef
```

Comment:
This is missing IMAGE_SIZE, should be 7808k ...

Reply:
`IMAGE_SIZE` is obsoleted by `TPLINK_FLASHLAYOUT`

Reply:
Well, while it looks that IMAGE_SIZE is not evaluated here, this would be the only device without it in entire ramips then. I would prefer to have it there for now, until we find a way how to deal with firmware partition size in the future.

Reply:
Done